client/blocks/subscription-length-picker/index.jsx

```diff
@@ -42,11 +47,37 @@ export class SubscriptionLengthPicker extends React.Component {
 		checked: this.props.initialValue,
 	};
 
+	formatTax( taxRate, price, currencyCode ) {
+		const { translate } = this.props;
+
+		// taxRate unknown
+		if ( ! isNumber( taxRate ) ) {
+			return translate( '+tax', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **54** times:
`translate( 'Tax' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).